### PR TITLE
added Back To Top button

### PR DESCRIPTION
**Pull request by LalitNM**

### CODE REVIEW

**Comment by LalitNM** [comment]
@Smokyish! anything else to do?

File: assets/css/main.css [hunk 1]
```diff
@@ -4245,3 +4245,12 @@ a.comms_guide_res_table strong {
   max-width: 40%;
   margin-bottom: 1em;
 }
+
+#BackToTop {
+  bottom: 10vh;
```
Comment (Smokyish) [inline assets/css/main.css]: Change to `bottom: 2.5vh`

File: assets/css/main.css [hunk 2]
```diff
@@ -4245,3 +4245,12 @@ a.comms_guide_res_table strong {
...
+  right: 10vw;
```
Comment (Smokyish) [inline assets/css/main.css]: Change to `right: 1.05vw;`

File: assets/css/main.css [hunk 3]
```diff
@@ -4245,3 +4245,12 @@ a.comms_guide_res_table strong {
...
+  position: fixed;
+  z-index: 999;
+  border: none;
+  border-radius: 50%;
+}
```
Comment (Smokyish) [inline assets/css/main.css:4258]: Adding padding fixes the oval shape, i.e., `padding: 2.5em 2.5em;`, but this causes an issue with the arrow positioning which should then be fixed.

Also the size should be decreased a little bit, i.e., `font-size: 9px;` with the above mentioned padding would look good size wise.

image

**Review by Smokyish — CHANGES_REQUESTED** [review]
Requesting some changes for the button.

The button appears as an oval shape instead of a sphere as requested in https://github.com/trustlines-protocol/tl-docs/issues/3
image

The position should also be lower and more to the right so the button fits better into the right side margin of the pages.

Lastly, this PR adds the button only to the `Home` page, because it is implemented in the `banner.html` which only appears in the `Home` page. To fix this and make the button appear consistently on all pages, it should instead be added to the `_includes/head.html` file, after the `</head>` element.

**Comment by Smokyish** [comment]
@LalitNM Thank you again for another PR! I've requested some changes on this one with the issues mentioned in the review.

File: assets/css/main.css (hunk 3 above)
Comment (LalitNM) [reply assets/css/main.css:4258]: I have done requested changes.
Now the button looks like:
button
Is this fine?

File: assets/css/main.css (hunk 1 above)
Comment (LalitNM) [reply assets/css/main.css]: Changed bottom distance.

File: assets/css/main.css (hunk 2 above)
Comment (LalitNM) [reply assets/css/main.css]: changed right distance. Now it is 1.5vw

**Comment by LalitNM** [comment]
@Smokyish I did requested changes. Let me know if something else has to de done.
I changed the character of button because that character was not fitting in the circle. New character fits there.

**Review by Smokyish — APPROVED** [review]
Looks good and works.

**Comment by Smokyish** [comment]
@LalitNM Thank you for another good PR! Could you again also do the same for https://github.com/trustlines-protocol/tl-dev-docs please? :)

**Pull request merged**